.gitignore

```
@@ -0,0 +1,14 @@
+.DS_Store
+.idea
+
+.ddev/*
+!.ddev/config.yaml
+!.ddev/docker-compose.environment.yaml
+
+/public/index.php
+/public/typo3temp
+/public/typo3conf/PackageStates.php
+/public/typo3conf/ENABLE_INSTALL_TOOL
```

That's not korrekt. We do not nee public/typo3conf/ext in git.

Please have a look at https://github.com/slub/ddev-slub-katalog/blob/main/.gitignore

```suggestion
/public/*
!/public/.htaccess
!/public/fileadmin/.gitkeep
!/public/typo3conf
/public/typo3conf/*
!/public/typo3conf/LocalConfiguration.php
!/public/typo3conf/AdditionalConfiguration.php
```